tests/genome/test_binning.py

```
@@ -75,3 +77,40 @@ def test_unitem_profile(test_temp: Path):
         .to_dict()
     )
     test_files / "binny_unitem_unanimous.tsv"
+
+
+@temp_output
+def test_smk_filter__rename_filtered_ls_tsv(test_temp: Path):
```

Ruff (0.11.9)

Redefinition of unused `test_temp` from line 16

(F811)